---
layout: default
title: Publications | Chen lab
description: The actual lab!
keywords: genomics, bacterial genetics, urinary tract infections, UPEC, synthetic biology, scicomm
url: https://swainechen.github.io/publications.html
custom_css: index.css
custom_js: responsive.js
---

<!-- Header -->

<div class="header home-header">
    <div class="Grid container">
        <div id="header-text" class="Grid-cell u-size7of8">
            <h2 class="small-title">Chen Lab Publications</h2>
        </div>
    </div>
</div>

<!-- Publications -->

<div class="section grey">
    <div class="Grid container">
        <div id="vision-text" class="Grid-cell">
              <h3 class="center-text large-margin">
                For a full list of publications, see <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=chen+swaine">PubMed</a> or <a href="https://orcid.org/0000-0002-0107-2861">ORCID</a>.<br>
                A few key publications are highlighted below, with some context.
              </h3>
              <hr width=40%>
        </div>
    </div>

    <div class="Grid small-container">
      <div class="Grid-cell Paper-card u-size1of3">
        <a href="https://www.ncbi.nlm.nih.gov/pubmed/31246981"><img src="assets/PubMed-logo.png" class="PubMed-icon"></a>
        Barkham T, Zadoks RN, Azmai MNA, Baker S, Bich VTN, Chalker V, Chau ML, Dance D, Deepak RN, van Doorn HR, Gutierrez RA, Holmes MA, Huong LNP, Koh TH, Martins E, Mehershahi K, Newton P, Ng LC, Phuoc NN, Sangwichian O, Sawatwong P, Surin U, Tan TY, Tang WY, Thuy NV, Turner P, Vongsouvath M, Zhang D, Whistler T, Chen SL (2019). "One hypervirulent clone, sequence type 283, accounts for a large proportion of invasive Streptococcus agalactiae isolated from humans and diseased tilapia in Southeast Asia." PLoS Negl Trop Dis. 2019 Jun 27;13(6):e0007421. doi: 10.1371/journal.pntd.0007421.
      </div>
      <div class="Grid-cell Context-card u-size2of3">
        In follow up work after the 2015 outbreak of GBS in Singapore (see Kalimuddin, et al. 2017 below), we asked whether GBS could have been causing disease prior to the outbreak and in countries outside of Singapore. The answer to both was, unfortunately, a resounding yes. We found evidence of similar GBS disease, caused by the same clone (ST283), throughout Southeast Asia for over 20 years. It may go back further, but this was as far back as we could find isolates. We have a strong suspicion that this disease has also been foodborne, possibly through eating raw freshwater fish, over these past 20 years as well, but this will require further studies.
      </div>
    </div>

    <hr width=40%>

    <div class="Grid small-container">
      <div class="Grid-cell Paper-card u-size1of3">
        <a href="https://www.ncbi.nlm.nih.gov/pubmed/30541067"><img src="assets/PubMed-logo.png" class="PubMed-icon"></a>
        Chen SL (2019). "Implementation of a Stirling number estimator enables direct calculation of population genetics tests for large sequence datasets." Bioinformatics. 2019 Aug 1;35(15):2668-2670. doi: 10.1093/bioinformatics/bty1012.
      </div>
      <div class="Grid-cell Context-card u-size2of3">
        We discovered that Fu's Fs statistic could be a powerful way to pinpoint the mutations that lead to emerging virulence (see Wu, et al. 2016 below). The calculation of Fu's Fs is built on Warren Ewen's famous Sampling Formula, which brings in some interesting mathematics; one of these is Stirling numbers of the first kind. As sequence datasets grow larger with genomics, the traditional methods for calculating these Stirling numbers lead to overflow issues even on modern computers. For other technical reasons, calculating Fu's Fs is also prone to underflow errors. I fixed these issues using an asymptotic estimator for Stirling numbers (amenable to calculation as logarithms) and alternative calculation routes for avoid underflow. These fixes not only extend the range of parameters for which we can calculate Fu's Fs, but they also dramatically increase calculation speed. The code for these can be found on my <a href="https://github.com/swainechen/hfufs">GitHub</a> page.
      </div>
    </div>

    <hr width=40%>

    <div class="Grid small-container">
      <div class="Grid-cell Paper-card u-size1of3">
        <a href="https://www.ncbi.nlm.nih.gov/pubmed/29378794"><img src="assets/PubMed-logo.png" class="PubMed-icon"></a>
        Duraiswamy S, Chee JLY, Chen S, Yang E, Lees K, Chen SL (2018). "Purification of Intracellular Bacterial Communities during Experimental Urinary Tract Infection Reveals an Abundant and Viable Bacterial Reservoir." Infect Immun. 2018 Mar 22;86(4). pii: e00740-17. doi: 10.1128/IAI.00740-17.
      </div>
      <div class="Grid-cell Context-card u-size2of3">
        This paper adapted a classical technique - mouth pipetting for single cell isolation - to the UTI field. For the first time, we were able to isolate pure populations of bladder epithelial cells infected (on the inside) with E. coli!This dovetails with the current breakneck progress in single cell genomics to finally enable the first genome-wide molecular view into intracellular infection during UTI. Not only that, it provides a route to the first single cell, niche-specific, simultaneous host + pathogen functional genomics assay.
      </div>
    </div>

    <hr width=40%>

    <div class="Grid small-container">
      <div class="Grid-cell Paper-card u-size1of3">
        <a href="https://www.ncbi.nlm.nih.gov/pubmed/28684862"><img src="assets/PubMed-logo.png" class="PubMed-icon"></a>
        Chen S, Larsson M, Robinson RC, Chen SL (2017). "Direct and convenient measurement of plasmid stability in lab and clinical isolates of E. coli." Sci Rep. 2017 Jul 6;7(1):4788. doi: 10.1038/s41598-017-05219-x.
      </div>
      <div class="Grid-cell Context-card u-size2of3">
        This manuscript described an additional application of our negative seleciton system for understanding bacterial plasmid stability. Plasmids are important for biotechnology and also are major contributors to the current antibiotic resistance crisis. Plasmids can carry many (>10) antibiotic resistance genes, and they can transfer between bacteria (even across species) - thus a single plasmid transfer event can immediately confer resistance to multiple antibiotic classes to a new bacterium. In addition to being able to transfer horizontally, plasmids have other systems that help to maintain them in their bacterial hosts. These systems represent potential targets to counteract plasmid stability and thereby reduce antibiotic resistance! In order to realize this potentially new anti-plasmid strategy, we need better tools to study the stability of plasmids, especially the large antibiotic resistance plasmids circulating in clinical bacterial isolates. This paper leveraged our negative selection system, which was designed for clinical isolates of Enterobacteriaceae, to develop multiple convenient assays to measure plasmid loss, a previously tedious task.
      </div>
    </div>

    <hr width=40%>

    <div class="Grid small-container">
      <div class="Grid-cell Paper-card u-size1of3">
        <a href="https://www.ncbi.nlm.nih.gov/pubmed/28475781"><img src="assets/PubMed-logo.png" class="PubMed-icon"></a>
        Kalimuddin S, Chen SL, Lim CTK, Koh TH, Tan TY, Kam M, Wong CW, Mehershahi KS, Chau ML, Ng LC, Tang WY, Badaruddin H, Teo J, Apisarnthanarak A, Suwantarat N, Ip M, Holden MTG, Hsu LY, Barkham T; Singapore Group B Streptococcus Consortium (2017). "2015 Epidemic of Severe Streptococcus agalactiae Sequence Type 283 Infections in Singapore Associated With the Consumption of Raw Freshwater Fish: A Detailed Analysis of Clinical, Epidemiological, and Bacterial Sequencing Data." Clin Infect Dis. 2017 May 15;64(suppl_2):S145-S152. doi: 10.1093/cid/cix021.
      </div>
      <div class="Grid-cell Context-card u-size2of3">
        This was the main manuscript describing a remarkable outbreak of foodborne Group B Streptococcus infections in Singapore in 2015. My lab contributed to the genomics analysis that helped to prove that the invasive GBS infections seen in otherwise healthy adults came from the raw fish that they ate. This was the first time that foodborne invasive GBS infections had been reported. Furthermore, this outbreak was also unique because it affected an atypical patient population - the infected adults in this outbreak were healthier, with fewer comorbidities, than those that are typically associated with invasive (non-foodborne) GBS disease.
      </div>
    </div>

    <hr width=40%>

    <div class="Grid small-container">
      <div class="Grid-cell Paper-card u-size1of3">
        <a href="https://www.ncbi.nlm.nih.gov/pubmed/27601641"><img src="assets/PubMed-logo.png" class="PubMed-icon"></a>
        Wu Z, Periaswamy B, Sahin O, Yaeger M, Plummer PJ, Zhai W, Shen Z, Dai L, Chen SL*, Zhang Q* (2016). "Point mutations in the major outer membrane protein drive hypervirulence of a rapidly expanding clone of Campylobacter jejuni." Proc Natl Acad Sci U S A. 2016 Sep 20;113(38):10690-5. (* Co-corresponding authors)
      </div>
      <div class="Grid-cell Context-card u-size2of3">
        This manuscript reported one major result: mutations in a single gene are responsible for the ability of one clone of C. jejuni to cause abortion in sheep. This manuscript had two major advances: the first was the use of sexual genetics (hybridization of two phenotypically different strains of Campylobacter followed by selection), which led to the identification of the porA gene as being responsible for abortion. The second advance was utilizing this knowledge to develop a purely computational genomics analysis that would identify porA from genome sequences alone.
      </div>
    </div>

    <hr width=40%>

    <div class="Grid small-container">
      <div class="Grid-cell Paper-card u-size1of3">
        <a href="https://www.ncbi.nlm.nih.gov/pubmed/27035967"><img src="assets/PubMed-logo.png" class="PubMed-icon"></a>
        Zhang H, Susanto TT, Wan Y, Chen SL (2016). "Comprehensive mutagenesis of the fimS promoter regulatory switch reveals novel regulation of type 1 pili in uropathogenic Escherichia coli." Proc Natl Acad Sci U S A, 2016 Apr 12; 113(15):4182-7.
      </div>
      <div class="Grid-cell Context-card u-size2of3">
        This manuscript describes a new genomics technique for studying the regulation of type 1 pili in uropathogenic E. coli (UPEC). Type 1 pili are the most important virulence factor that UPEC require for causing urinary tract infection (UTI). They are expressed from a single promoter (denoted fimS) which can be turned on and off by physical inversion (i.e. fimS is cut out and pasted back into the chromosome in the opposite orientation). The inversion is regulated and depends on the local sequence context of fimS. Using our negative selection system (see Khetrapal, et al. below), we made a comprehensive library of mutations in and near fimS, preserving its native context, in a clinical UPEC strain. Custom analysis of third generation PacBio sequencing simultaneously identified mutations and associated them with the quantitative fimS orientation (individual bacteria are either on or off, but a population may be between 0 to 100% on). We identified known regulatory sequences in fimS and also discovered a previously unknown regulatory element adjacent to fimS.
      </div>
    </div>

    <hr width=40%>

    <div class="Grid small-container">
      <div class="Grid-cell Paper-card u-size1of3">
        <a href="https://www.ncbi.nlm.nih.gov/pubmed/26447926"><img src="assets/PubMed-logo.png" class="PubMed-icon"></a>
        Eshaghi M, Sun G, Grüter A, Lim CL, Chee YC, Jung G, Jauch R, Wohland T, Chen SL (2015). "Rational Structure-Based Design of Bright GFP-Based Complexes with Tunable Dimerization." Angew Chem Int Ed Engl. 2015 Oct 8. doi: 10.1002/anie.201506686.
      </div>
      <div class="Grid-cell Context-card u-size2of3">
        This manuscript describes a modified GFP, which we termed "vGFP". The vGFP design consists of a fusion between GFP and a GFP-binding protein that enhances the brightness of GFP by ~50%. We predicted that adjusting how GFP and the binding protein were linked would change whether the fusion formed a monomer or dimer, which was verified with biochemistry, crystallography, and single molecule fluorescence. The vGFP design helped us to create reporter strains of uropathogenic E. coli (UPEC) that were 10× brighter than existing strains used in the urinary tract infection (UTI) field with no defects in virulence.
        <p><p>
        Plasmids with the vGFP constructs are available at <a href="https://www.addgene.org/Swaine_Chen/">AddGene</a>. Please contact <a href="mailto:slchen_AT_gis.a-star.edu.sg">Swaine</a> if the fluorescent UTI89 strains would be useful for your work.
      </div>
    </div>

    <hr width=40%>

    <div class="Grid small-container">
      <div class="Grid-cell Paper-card u-size1of3">
        <a href="https://www.ncbi.nlm.nih.gov/pubmed/26447926"><img src="assets/PubMed-logo.png" class="PubMed-icon"></a>
        Khetrapal V, Mehershahi K, Rafee S, Chen S, Lim CL, Chen SL (2015). "A set of powerful negative selection systems for unmodified Enterobacteriaceae." Nucleic Acids Research pii: gkv248.
      </div>
      <div class="Grid-cell Context-card u-size2of3">
        This manuscript describes a modular set of negative selection cassettes designed for wild-type, disease-causing clinical isolates of bacteria. Antibiotic resistance genes are an example of positive selection; bacteria carrying a resistance gene survive when treated with antibiotic. Negative selection is the opposite; bacteria carrying the cassette will die when placed in the appropriate conditions. Positive and negative selection together enable "perfect" manipulation of the bacterial genome, where arbitrary mutations can be made with no extraneous marker mutations. While positive selection has generally been available for all bacteria, negative selection systems have been designed for and tested in cloning strains of E. coli. Our system enables negative selection in a broad range of bacteria including E. coli and Salmonella. Of note, in cloning strains of E. coli, our system is 60× better (measured as killing of bacteria carrying the negative selection cassette) than the next best published system and 10,000× better than the most common system in use. In clinical strains, ours is the only generally usable system.
        <p><p>
        These selection plasmids are available at <a href="https://www.addgene.org/Swaine_Chen/>AddGene</a>.
      </div>
    </div>

</div>
